test/test.admin-myevents.selenium.js

```diff
@@ -0,0 +1,174 @@
+var expect = require('expect.js'),
+    common = require('./common'),
+    mandrill = require("mandrill-api"),
+    outbox = [];
+
+describe("ADMIN MYEVENTS SELENIUM", function() {
+    var browser = null,
+        event = null;
+
+    if (process.env.SKIP_SELENIUM_TESTS) {
+        return;
+    }
+    this.timeout(60000); // Extra long timeout for selenium :(
+
+    before(function(done) {
+        this.timeout(240000);
+
+        /* Mock Mandrill's API here */
+        mandrill.Mandrill = function(apiKey) {
+            this.messages = {
+                send: function(messageObj) {
+                    outbox.push(messageObj.message);
+                }
+            }
+        };
+
+        common.getSeleniumBrowser(function (theBrowser) {
+            browser = theBrowser;
+            common.standardSetup(done);
+        });
+    });
+
+    after(function(done) {
+        browser.quit().then(function() {
+            common.standardShutdown(done);
+        });
+    });
+
+    it("Can't access myevents page unauthenticated", function(done) {
+        browser.get(common.URL + "/myevents/");
+        browser.getCurrentUrl().then(function(url) {
+            expect(url).to.not.eql(common.URL + "/myevents/");
+            done();
+        });
+    });
+
+    it("Can't access myevents page if neither a superuser nor an admin of event", 
```

Comment:
What I meant was: add a test that checks the fix you made to admin-hood.  We don't want to accept a post from an admin to an event they are not an admin of.  In short: this test currently checks "are you an admin of this event", which is great.  One more iteration of almost the same test would check "are you an admin of some event, but not this one."